app/controllers/application_controller.rb

```diff
@@ -112,4 +118,16 @@ def pagination_params
   def render_job_id(jid)
     render json: { job_id: jid }, status: 202
   end
+
+  def allowed_verbs
+    all_methods = ActionDispatch::Routing::HTTP_METHODS
```

Comment:
consider using this directly in the call below or maybe using a private method, doesn't appear as the local variable is used more than once, here.